documentation/src/main/asciidoc/topics/proc_server_using_datasources.adoc

```diff
@@ -15,19 +15,19 @@ configuration of your cache configuration, as in the following example:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<distributed-cache-configuration name="persistent-cache">
+<distributed-cache-configuration name="persistent-cache" xmlns:jdbc="urn:infinispan:config:store:jdbc:{schemaversion}">
     <persistence>
-        <string-keyed-jdbc-store>
-            <data-source jndi-url="jdbc/datasource"/> <1>
-            <string-keyed-table drop-on-exit="true"
+        <jdbc:string-keyed-jdbc-store>
+            <jdbc:data-source jndi-url="jdbc/postgres"/>
+            <jdbc:string-keyed-table drop-on-exit="true"
                           create-on-start="true"
                           prefix="TBL">
-                <id-column name="ID" type="VARCHAR(255)"/>
-                <data-column name="DATA" type="BINARY"/>
-                <timestamp-column name="TS" type="BIGINT"/>
-                <segment-column name="S" type="INT"/>
-            </string-keyed-table>
-        </string-keyed-jdbc-store>
+                <jdbc:id-column name="ID" type="VARCHAR(255)"/>
+                <jdbc:data-column name="DATA" type="BYTEA"/>
```

Comment:
is `BYTEA` a valid type? (honest question, I don't know the available SQL types :))

Reply:
@pruivo yeah I believe it is. https://www.postgresql.org/docs/7.2/datatype-binary.html 

I'm not the expert here though. I'll double check this with Gustavo to be sure.

Reply:
@oraNod @pruivo depends on the database.
For the data column type we have:
postgres: BYTEA
mysql, mariadb, mssql: VARBINARY
db2: blob
oracle: raw
sybase: image